tools/lldb-moduleimport-test/lldb-moduleimport-test.cpp

```diff
@@ -254,6 +257,10 @@ int main(int argc, char **argv) {
   Invocation.setModuleName("lldbtest");
   Invocation.getClangImporterOptions().ModuleCachePath = ModuleCachePath;
 
+  if (!ResourceDir.empty()) {
+    Invocation.setRuntimeResourcePath(ResourceDir);
```

Comment:
Do you need this for testing? [If so, can you add a test for this].

Reply:
It's for debugging: I often use a Debug compiler with a standard library that was build with a Release+Asserts compiler, because the latter builds the standard library so much faster. Testing doesn't need this.

Reply:
OK, makes sense, thanks!